package.json

```diff
@@ -50,6 +52,7 @@
   },
   "devDependencies": {
     "@rubensworks/eslint-config": "^2.0.0",
+    "@types/babel__parser": "^7.1.1",
```

Comment:
Sometime I'm having an error saying that the type description is missing to I added it to the dev dependencies.

Reply:
Can you check how to fix this on your system? Because this devdep should not be needed here.